site/en/blog/blinkng/index.md

```diff
@@ -0,0 +1,185 @@
+---
+layout: 'layouts/blog-post.njk'
+title: 'RenderingNG deep-dive: BlinkNG'
+description: >
+  Find out about BlinkNG and the various sub-projects that addressed long-standing problems in the organization and structure of Blink code.
+thumbnail: image/kheDArv5csY6rvQUJDbWRscckLr1/G8miv1svP6yMEZbP0DqX.png
+alt: The rendering pipeline before and aftr BLinkNG.
+date: 2022-03-29
+authors:
+  - stefanzagar
+tags:
+  - rendering
+---
+{% Aside %}
+This post is a part of a series on the Chromium rendering engine. Check out the rest of the series to learn more about [RenderingNG](/blog/renderingng/), [RenderingNG architecture](/blog/renderingng-architecture/), [key data structures](/blog/renderingng-data-structures/), [VideoNG](/blog/videong/), and [LayoutNG](/blog/layoutng/).
+{% endAside %}
+
+
+_Blink_ refers to Chromium's implementation of the [web platform](https://en.wikipedia.org/wiki/Web_platform), and it encompasses all the phases of rendering prior to compositing, culminating in _compositor commit_. You can read more about blink rendering architecture in a previous [article](/blog/renderingng-architecture/#render-process-main-thread-components) in this series. 
+
+[Blink](https://en.wikipedia.org/wiki/Blink_(browser_engine)) began life as a fork of [WebKit](https://en.wikipedia.org/wiki/WebKit), which is itself a fork of [KHTML](https://en.wikipedia.org/wiki/KHTML), which dates to 1998. It contains some of the oldest (and most critical) code in Chromium, and by 2014 it was definitely showing its age. In that year, we embarked on a set of ambitious projects under the banner BlinkNG, with the goal of addressing long-standing deficiencies in the organization and structure of the Blink code. This article will explore BlinkNG and its constituent projects: why we did them, what they accomplished, the guiding principles that shaped their design, and the opportunities for future improvements they afford.
+
+{% Img src="image/kheDArv5csY6rvQUJDbWRscckLr1/G8miv1svP6yMEZbP0DqX.png", alt="The rendering pipeline before and after BlinkNG.", width="800", height="362" %}
```

Comment:
I think that would have been @una

Reply:
I'll follow up with Una, thanks.

Reply:
![Untitled_Artwork](https://user-images.githubusercontent.com/1693164/163468667-ab3221bb-d2c9-4540-b1df-5e0f33312fc1.jpeg)

Attached the illustration, still awaiting feedback before coloring :)

Reply:
This looks good to me, my only thought is that it would be fun if you could somehow give a subtle nod to this meme template:

https://memegen.corp.google.com/384765

Reply:
Updated illustration:
![EA095E8E-5C1D-4A0D-962B-B6813C7FD93A](https://user-images.githubusercontent.com/1693164/163605152-05feae67-6027-4b06-8883-6f889ca527c4.png)

d.c.c upload url:
```
{% Img src="image/HodOHWjMnbNw56hvNASHWSgZyAf2/eg6i9S7UHWfrkF9Oube7.png", alt="ALT_TEXT_HERE", width="800", height="391" %}
```

Reply:
Looks good to me also, let's add the image and publish.